packages/aic/src/low_pid.ts

```diff
@@ -0,0 +1,28 @@
+import { pid } from './pid'
+
+const run = async () => {
```

Comment:
I don't think we want this in `src`

I made a separate `bench` command in `common` that runs through jest. may be good to model it off of